Application/EdFi.Ods.Common/Caching/CacheInterceptor.cs

```diff
@@ -0,0 +1,58 @@
+﻿// SPDX-License-Identifier: Apache-2.0
+// Licensed to the Ed-Fi Alliance under one or more agreements.
+// The Ed-Fi Alliance licenses this file to you under the Apache License, Version 2.0.
+// See the LICENSE and NOTICES files in the project root for more information.
+
+using System.Reflection;
+
+namespace EdFi.Ods.Common.Caching;
```

Comment:
Can you consolidate the using statements

Reply:
Done.